ui/lib/ldap/addon/routes/roles/role/details.ts

```diff
@@ -0,0 +1,37 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+
+import Route from '@ember/routing/route';
+
+import type LdapRoleModel from 'vault/models/ldap/role';
+import type Controller from '@ember/controller';
+import type Transition from '@ember/routing/transition';
+import type { Breadcrumb } from 'vault/vault/app-types';
+
+interface LdapRoleDetailsController extends Controller {
+  breadcrumbs: Array<Breadcrumb>;
+  model: LdapRoleModel;
+}
+
+export default class LdapRoleEditRoute extends Route {
+  model() {
+    return this.modelFor('roles.role');
+  }
```

Comment:
I think this model hook can be removed since ember magically passes down the parent's model